src/new-client/src/components/DraggableTearOut/DraggableTearOut.tsx

```diff
@@ -0,0 +1,112 @@
+import styled from "styled-components"
+import { RefObject, useState } from "react"
+import { tearOutSection, Section } from "@/App/TearOutSection/state"
+import { canDrag } from "@/components/DraggableTearOut/canDrag"
+import { tearOut } from "@/App/LiveRates/Tile/TearOut/state"
+
+export const DragWrapper = styled.div`
+  height: 100%;
+`
+
+type DraggableSectionTearOutProps = React.PropsWithChildren<{
+  section: Section
+  disabled?: boolean
+}>
+
+type DraggableTileTearOutProps = React.PropsWithChildren<{
+  symbol: string
+  tileRef: RefObject<HTMLDivElement>
+  disabled?: boolean
+}>
+
+type DraggableTearOutGenericProps = React.PropsWithChildren<{
+  dragHandler: () => void
+}>
+
+export const DraggableSectionTearOut: React.FC<DraggableSectionTearOutProps> = (
+  props,
+) => {
+  const draggable = canDrag && !props.disabled
+  const dragHandler = () => {
+    tearOutSection(true, props.section)
+  }
+  return draggable ? (
+    <DraggableTearOut dragHandler={dragHandler}>
+      {props.children}
+    </DraggableTearOut>
+  ) : (
+    <>props.children</>
```

Comment:
Missing `{}` here, it's just a string at the moment.
Also, can just be `: ( props.children ) `

Reply:
For some reason TS doesn't like this notation, for now I added a `@ts-ignore`, will investigate further.

Reply:
I couldn't work out why so I checked out locally.
`React.PropsWithChildren` adds children as an optional prop, so TS doesn't like `: ( props.children )` because it could be undefined, which would not satisfy the `React.FC` type.
Options are to turn (trick) it into thinking it's a component with the `<>{props.children}</>` like you had, or to change the props to make children mandatory. 
`<>{props.children}</>` is probably the simplest thing to do.

Reply:
Done!